akka-docs/src/main/paradox/stream/operators/Source/range.md

```diff
@@ -17,6 +17,7 @@ Emit each integer in a range, with an option to take bigger steps than 1.
 
 Emit each integer in a range, with an option to take bigger steps than 1. @scala[In Scala, use the `apply` method to generate a sequence of integers.]
 
+## Reactive Streams semantics
```

Comment:
this one has Examples, and then I would prefer to have Examples before the sematics, because Examples are more useful

Reply:
I agree, but figured that'd be something for another day :)